explorer/frontend/__tests__/api/stats.test.js

```diff
@@ -51,8 +51,9 @@ const runStatsTest = async (functionToTest, args, responseMap, expectedResult) =
 	spy.mockImplementation(url => {
 		const response = responseMap[url];
 
-		if (response) return Promise.resolve(response);
-		else
+		if (response) {
+			return Promise.resolve(response);
+		} else
```

Comment:
Prettier doesn't accept the code format without either making it a single line or adding brackets. Here's the answer from the Prettier team: https://github.com/prettier/prettier/issues/366#issuecomment-302962358

To address this, I've removed the Prettier plugin from the ESLint configuration and moved Prettier execution to a separate script to avoid conflicts. As a result, there are new changes in some files after running Prettier and ESLint separately.